src/main/webapp/app/tasks/default-task.service.ts

```diff
@@ -16,10 +16,18 @@ export class DefaultTaskService implements TaskService {
     return this.http.post<Task>(this.baseUrl + '/tasks', {name} as Task);
   }
 
+  update(id: string): Observable<Task> {
+    return this.http.post<Task>(this.baseUrl + '/tasks', {id} as Task);
+  }
+
   delete(id: string): Observable<void> {
     return this.http.delete<void>(this.baseUrl + '/tasks/' + id);
   }
 
+  clearCompletedTasks(): Observable<Task[]> {
+    return this.http.get<Task[]>(this.baseUrl + '/tasks');
```

Comment:
Why is the clear completed tasks implemented as get, i guess this should be a delete.